OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/AdminControllerTests.cs

```diff
@@ -192,33 +191,18 @@ public async Task GetApplications_WhenCollectionIsEmpty_ShouldReturnNoContent()
         // Arrange
         controller.ControllerContext.HttpContext = fakeHttpContext;
         controller.ControllerContext.HttpContext.SetContextUser(Role.TechAdmin);
-        sensitiveApplicationService.Setup(s => s.GetAll(It.IsAny<ApplicationFilter>())).ReturnsAsync(new SearchResult<ApplicationDto>()
+        sensitiveApplicationService.Setup(s => s.GetAll(It.IsAny<ApplicationFilter>())).ReturnsAsync(new SearchResult<ApplicationDto>
         {
-            Entities = new List<ApplicationDto>(),
-            TotalAmount = 0,
+            Entities = applications,
+            TotalAmount = applications.Count,
         });
 
         // Act
-        var result = await controller.GetApplications(new ApplicationFilter()).ConfigureAwait(false) as NoContentResult;
+        var result = await controller.GetApplications(new ApplicationFilter()).ConfigureAwait(false) as OkObjectResult;
 
         // Assert
         result.Should().NotBeNull();
-        result.StatusCode.Should().Be(StatusCodes.Status204NoContent);
-    }
-
-    [Test]
-    public async Task GetApplications_WhenCalledParentOrProvider_RetursObjectResult()
```

Comment:
This test was removed. Please give more details why.

Reply:
> This test was removed. Please give more details why.

I did it, because in ```AdminController``` one if statement was remowed (checking whether user is ```TechAdmin```). That it why I thought no cheking, no test for it))
Thank you for the comments

Reply:
Thank you for explanation, my fault that I didn't notice it at first review. Resolved